cve_bin_tool/checkers/sqlite.py

```diff
@@ -36,6 +38,12 @@ def guess_version(lines, version_map):
             if mapping[1] in line:
                 return mapping[0]
 
+    # if all else fails, use the vague pattern
+    version = regex_find(lines, *vague_pattern)
+    if version:
+        if len(version) == 1:
+            return "{}.UNKNOWN".format(version)
```

Comment:
Why is this string being formatted? I've been away for a while so I'm not sure what's going on here, but what's the intention here?